test/e2e/generator/generate.go

```diff
@@ -16,6 +16,7 @@ var (
 	testnetCombinations = map[string][]interface{}{
 		"topology":      {"single", "quad", "large"},
 		"ipv6":          {false, true},
+		"useNewP2P":     {false, true, 2, 3},
```

Comment:
What's with the `2, 3` part?

Reply:
for these networks, make a legacy network but have half or a third of the nodes use the new stack.

Reply:
also any field in this map gets systematically tested of all permutations so you'll need to be careful about how many you add. I think prior we generated around 90 testnets for the nightlies. With this addition we will have around 360 testnets

Reply:
> What's with the `2, 3` part? 

Ah this is to account for mixed testnets

Reply:
cool, I think it makes sense to test "on, off, half" suites, but not "third", which cuts back some? 

How long does a single testnet run take?

Reply:
I think I would just have three combinations: "new", "old", "split"

Reply:
> How long does a single testnet run take?

it varies quite a bit depending on the size and the amount of perturbations. I would say between 3 - 20 seconds

Reply:
I think we 're on the same page.